### PR TITLE
Issue 1 Work

### PR DESCRIPTION
**Pull request by pvmoura**
I added a Sidekiq job to email suspended members

### CODE REVIEW

**Comment by flah00** [comment]
I'm so sorry I didn't see this sooner. For some reason, I did not see the notice of the PR. I will review this ASAP!

**Comment by flah00** [comment]
Could you clean up the white spacing? Let's try to keep it two spaces for everything in the repo.

**Comment by pvmoura** [comment]
I updated the white spacing and will use 2 spaces in the future. Also, I haven't done that much professional work in Rails, so please critique liberally. Thanks!

File: app/jobs/email_suspended_members.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+class EmailSuspendedMembers
+  include Sidekiq::Worker
+  sidekiq_options queue: :members, retry: true, backtrace: true, failures: true 
+
+  def perform
+    suspended_members = Member.where(:status => "suspended")
```
Comment (flah00) [inline app/jobs/email_suspended_members.rb]: since we're using a newer version of ruby, hash keys can more readably be defined as `status: "suspended"`

**Comment by flah00** [comment]
generally speaking, forking and pushing PRs upstream can get messy. You always need to make sure that you pull your upstream remote, then rebase that onto your origin. If you're not comfortable with git, that can be cumbersome. If you simply create topic branches on bushwickbfc/bfc-members, it might be easier on you.

Right now, your origin is behind your upstream. So you're missing changes that are currently in production. See https://help.github.com/articles/syncing-a-fork/ for more details.

File: app/jobs/email_suspended_members.rb (hunk 1 above)
Comment (flah00) [inline app/jobs/email_suspended_members.rb]: it should be noted that this only applied to symbols. objects cannot use this notation.

**Comment by flah00** [comment]
merged

**Pull request closed without merging**